packages/preview/silky-letter-insa/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "silky-letter-insa"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["SkytAsul"]
+license = "MIT"
+description = "A template made for letters and short documents of INSA, a French engineering school."
+homepage = "https://github.com/SkytAsul/INSA-Typst-Template"
+repository = "https://github.com/SkytAsul/INSA-Typst-Template"
+keywords = ["insa", "school", "engineering", "letter"]
+disciplines = ["engineering", "computer-science", "mathematics", "physics"]
+categories = ["office", "flyer"]
```

Comment:
The category `flyer` doesn't really seem fitting. It is intended for compact leaflets.